aea/aea_builder.py

```diff
@@ -590,6 +604,8 @@ def build(self, connection_ids: Optional[Collection[PublicId]] = None) -> AEA:
         self._load_and_add_protocols()
         self._load_and_add_contracts()
         connections = self._load_connections(identity.address, connection_ids)
+        # add alrady laoded connections
```

Comment:
fix typo: `already loaded` instead of `alrady laoded`

Reply:
done.
agree with you, there is a difference in component construction.